Script/AtomicEditor/ui/frames/menus/ProjectFrameMenu.ts

```diff
@@ -101,6 +101,11 @@ class ProjectFrameMenus extends Atomic.ScriptObject {
                 return true;
             }
 
+            if (refid == "force_reimport_folder") {
+                ToolCore.assetDatabase.reimportAllAssetsInDirectory(path);
```

Comment:
"reimportDirectory" may be a better method name, does this have a recursive flag, I won't hold up the PR for this, but if there is something else might be a good idea to rename.  Also, noticing a trend here with manual scanning, maybe we need to hook up a file watcher on the project folders to catch changed assets for reimport at this point?  The file watcher will catch resource updates, like textures now, though I don't think we have it reimporting assets